test/functional/example_test.py

```diff
@@ -78,7 +78,10 @@ def set_test_params(self):
         """Override test parameters for your individual test.
 
         This method must be overridden and num_nodes must be explicitly set."""
-        self.setup_clean_chain = True
+        # By default the test pre-mines a chain of 200 blocks automatically.
```

Comment:
The test pre-mines, or it loads the cached data directories?

Reply:
Yeah, it loads a pre-mined chain. Given that mostly absolute beginners are going to look at this I was trying to make it easier to understand but in this case was giving the wrong impression. Updated.